src/platform/common/utils/localize.ts

```diff
@@ -848,20 +964,23 @@ export namespace DataScience {
     );
     export const noInterpreter = localize('DataScience.noInterpreter', 'No python selected');
     export const notebookNotFound = localize(
-        'DataScience.notebookNotFound',
-        'python -m jupyter notebook --version is not running'
+        {
+            key: 'DataScience.notebookNotFound',
+            comment: ["{Locked='version'}"]
+        },
+        'python -m Jupyter notebook --version is not running'
```

Comment:
I think `pyhton -m jupyter notebook` should be lcoked, and we shoudl not change the casing of `jupyter` to `Jupyter` as this is a CLI

Reply:
Ok doing this change! Thank you, it was my mistake